worker/ocspStatusWorker/ocspStatusWorker.go

```diff
@@ -0,0 +1,81 @@
+package ocspStatusWorker
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"crypto/x509"
+
+	"github.com/mozilla/tls-observatory/logger"
+	"github.com/mozilla/tls-observatory/worker"
+	"golang.org/x/crypto/ocsp"
+	"bytes"
+	"crypto"
+	"time"
+	"encoding/base64"
+)
+
+var workerName = "ocspStatusWorker"
+var workerDesc = `Determines a certificate's status via OCSP'`
+
+var log = logger.GetLogger()
+
+func init() {
+	worker.RegisterWorker(workerName, worker.Info{Runner: new(ocspStatusWorker), Description: workerDesc})
+}
+
+type ocspStatusWorker struct {
+	status OCSPStatus
+}
+
+type OCSPStatus struct {
+	Status int `json:"status"`
+	RevokedAt time.Time `json:"revoked_at"`
+
+}
+
+type params struct {
+}
+
+func (w ocspStatusWorker) Run(in worker.Input, resChan chan worker.Result) {
+	res := worker.Result{WorkerName: workerName, Success:false}
+
+	rawCert, _ := base64.StdEncoding.DecodeString(in.CertificateChain.Certs[0])
+	certificate, _ := x509.ParseCertificate(rawCert)
+	rawIssuerCert, _ := base64.StdEncoding.DecodeString(in.CertificateChain.Certs[1])
+	issuerCertificate, _ := x509.ParseCertificate(rawIssuerCert)
```

Comment:
You should check for errors here.

It might also be worthwhile to make sure the issuer is the right one (that it sign the end-entity) because ordering isn't always guaranteed. You can use CheckSignatureFrom from the x509 package for this.